store/sql_store.go

```diff
@@ -371,8 +381,10 @@ func (ss SqlStore) createIndexIfNotExists(indexName string, tableName string, co
 		}
 
 		query := ""
-		if fullText {
+		if indexType == INDEX_TYPE_FULL_TEXT {
 			query = "CREATE INDEX " + indexName + " ON " + tableName + " USING gin(to_tsvector('english', " + columnName + "))"
+		} else if indexType == INDEX_TYPE_PATTERN {
+			query = "CREATE INDEX " + indexName + " ON " + tableName + " (" + columnName + " text_pattern_ops)"
```

Comment:
why go through the hassle of using the db when it's such a small number of webhooks ??.  Just return the all and post filter in the server.  Since the first word is constantly changing you never get any db query cache.  You could just hold the hooks in memory.